spec/Overview.html

```diff
@@ -1328,8 +1328,7 @@ <h3>Data Types</h3>
           <h3>Methods and Parameters</h3>
           <p>
             Unless otherwise stated, objects created by the methods defined in this section shall be associated with the
-            [= relevant global object =]
-            of `this` [[HTML]].
+            [= relevant realm =] of [= this =].
```

Comment:
This is interesting — is an “ambient” realm argument like this typical @annevk?

Reply:
Not really, though there is an issue against IDL to make realm available as ambient information in a way that matches this as folks get it wrong all the time.